Examples/SnapinsSDKKotlinExample/README.md

```diff
@@ -0,0 +1,78 @@
+# Snap-ins SDK Kotlin Example Application
+
+
+An example implementation of the Snap-ins SDK using [Kotlin](https://kotlinlang.org/).
+
+See the [Developer Documentation](https://developer.salesforce.com/docs/atlas.en-us.noversion.service_sdk_android.meta/service_sdk_android/servicesdk_android_dev_guide.htm)
+for more detailed implementation instructions.
+
+
+## Required Configurations
+
+### Settings Activities
+
+The configuration values (organization id, deployment id, pod url, etc) for the snap-in SDKs can be
```

Comment:
Use just capital "s" for product name in paragraph: "...for the Snap-ins SDK..."